packages/storybook/src/stories/ino-checkbox/ino-checkbox.stories.ts

```diff
@@ -49,43 +58,17 @@ export const Playground: Story<Components.InoCheckbox> = (args) => html`
   </ino-checkbox>
 `;
 
-Playground.args = {
-  checked: false,
-  disabled: false,
-  indeterminate: false,
-  name: '',
-  selection: false,
-  value: '',
-};
+export const Playground = templateForComponent(Template, {});
 
-export const Variations = (): TemplateResult => html`
-  <div class="story-states">
-    <ino-checkbox checked>Checked</ino-checkbox>
-    <ino-checkbox>Unchecked</ino-checkbox>
-    <ino-checkbox indeterminate>Indeterminated</ino-checkbox>
-  </div>
-  <div class="story-states">
-    <ino-checkbox disabled checked>Checked and Disabled</ino-checkbox>
-    <ino-checkbox disabled>Unchecked and Disabled</ino-checkbox>
-    <ino-checkbox disabled indeterminate
-      >Indeterminate and Disabled
-    </ino-checkbox>
-  </div>
-`;
+export const Checked = templateForComponent(Template, { checked: true });
+export const Unchecked = templateForComponent(Template, { checked: false });
 
-export const Selection = (): TemplateResult => html`
-  <div class="story-states">
-    <ino-checkbox selection checked>Checked</ino-checkbox>
-    <ino-checkbox selection>Unchecked</ino-checkbox>
-    <ino-checkbox selection indeterminate>Indeterminate</ino-checkbox>
-  </div>
-  <div class="story-states">
-    <ino-checkbox checked selection disabled
-      >Checked and disabled
-    </ino-checkbox>
-    <ino-checkbox selection disabled>Unchecked and disabled</ino-checkbox>
-    <ino-checkbox selection indeterminate disabled>
-      Indeterminate Disabled
-    </ino-checkbox>
-  </div>
-`;
+// While checkboxes are mainly used in lists, the selection should be used as a single, independent UI element.
+export const Selection = templateForComponent(Template, {
```

Comment:
Good to know that this would be possible, but looks too complicated to me. I would keep the inheritance level on 1, meaning
```
export const Selection = templateForComponent(Checked, {indeterminate: true})
```

if possible

Reply:
I agree, it was only meant to show how it's done if we need it e.g. for some complex states